### PR TITLE
template delimiters

### PR DESCRIPTION
**Pull request by unai-programmfabrik**

### CODE REVIEW

File: main_test.go [hunk 1]
```diff
@@ -186,4 +188,59 @@ func testTemplateEvalCSVStdIn(t *testing.T) {
 	if string(out) != outputSampleText {
 		t.Fatalf("Expected:\n%s\nGot:\n%s", outputSampleText, string(out))
 	}
-}
\ No newline at end of file
+}
+
+func testTemplateDelimiters(t *testing.T) {
```
Comment (martinrode) [inline main_test.go:193]: Is your test running a binary? It shouldn't.

File: main.go [hunk 2]
```diff
@@ -104,7 +109,7 @@ func parseVariables(cmd *cobra.Command, args []string) {
 			SkipEmptyColumns: true,
 		}
 
-		templateCSVVariables[key], err = csvp.Untyped(csvBytes)
+		templateCSVVariables[key], err = csvp.Typed(csvBytes)
```
Comment (martinrode) [inline main.go:112]: Isnt this a big change here?Does this need to be part of this patch?

**Review by martinrode — CHANGES_REQUESTED** [review]

File: main.go (hunk 2 above)
Comment (unai-programmfabrik) [reply main.go:112]: The unit tests didn't pass because of it, we clearly use typed CSVs, no idea how it was in master

File: main_test.go (hunk 1 above)
Comment (unai-programmfabrik) [reply main_test.go:193]: Its not, its running a command the very same way the other tests above

**Pull request merged**